src/linttest/golden_test.go

```diff
@@ -225,9 +226,16 @@ func runGoldenTestsE2E(t *testing.T, targets []*goldenTest) {
 		return
 	}
 
+	var linterName string
+	if runtime.GOOS == "windows" {
+		linterName = "phplinter.exe"
+	} else {
+		linterName = "phplinter"
+	}
```

Comment:
I believe Linux doesn't care about binary file extension, so maybe we can always create a file with `.exe` extension?

Reply:
We may also want to add `phplinter.exe` to the gitignore file, just to be safe.